### PR TITLE
Bump copier

### PR DESCRIPTION
**Pull request by Santhin**
changes:
-   Bump packages
-   Drop support for `dbt2looker` due to bumping copier to `7.0.1`
---
Keep in mind:
- [x] Documentation updates
- [x] [Changelog](CHANGELOG.md) updates

### CODE REVIEW

File: data_pipelines_cli/cli_commands/create.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def create(project_path: str, template_path: Optional[str], vcs_ref: str) -> Non
     config = read_env_config()
     config_templates = config["templates"]
     src_template_path = _get_template_path(config_templates, template_path)
-    copier.copy(src_path=src_template_path, dst_path=project_path, vcs_ref=vcs_ref)
+    copier.run_auto(src_path=src_template_path, dst_path=project_path, vcs_ref=vcs_ref)
```
Comment (p-pekala) [inline data_pipelines_cli/cli_commands/create.py:57]: What is the difference?

File: tests/cli_commands/test_init.py [hunk 2]
```diff
@@ -29,15 +29,16 @@ class InitCommandTestCase(unittest.TestCase):
             ),
             "local-template": TemplateConfig(
                 template_name="local-template",
-                template_path="/var/tmp/Documents/project-template",
+                template_path="/Users/test_user/Documents/project-template",
```
Comment (p-pekala) [inline tests/cli_commands/test_init.py:32]: does that mean every developer will have "test_user" dir under Users? Why do you need this change?

**Review by p-pekala — CHANGES_REQUESTED** [review]

File: data_pipelines_cli/cli_commands/create.py (hunk 1 above)
Comment (Santhin) [reply data_pipelines_cli/cli_commands/create.py:57]: In previous releases, they were using `copy` and this changed to `run_auto` obviously they are still giving us backward compatibility but I was thinking of upgrading this as well.

example from their codebase
```
from .main import run_auto as copy  # noqa: F401; Backwards compatibility
```

File: tests/cli_commands/test_init.py (hunk 2 above)
Comment (Santhin) [reply tests/cli_commands/test_init.py:32]: `example_config_dict` contains data generated by user from [template](https://github.com/getindata/data-pipelines-cli/blob/bump-copier/tests/goldens/config_template/dp.yml.jinja), and in this test the answer for question `username` will be `test_user`. I kinda trimmed this to have only one question but the outcome is the same

**Review by p-pekala — APPROVED** [review]

**Pull request merged**